examples/client/client.c

```diff
@@ -2122,7 +2122,10 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
         wolfSSL_CTX_set_group_messages(ctx);
 
 #ifndef NO_DH
-    wolfSSL_CTX_SetMinDhKey_Sz(ctx, (word16)minDhKeyBits);
+    if (wolfSSL_CTX_SetMinDhKey_Sz(ctx, (word16)minDhKeyBits)
```

Comment:
John, we have two more cases of missing return code checks in `examples/benchmark/tls_bench.c` at lines 646 and 963. Would you mind patching those up as well? Thanks.

Reply:
Done.